worlds/Files.py

```diff
@@ -78,24 +82,15 @@ class InvalidDataError(Exception):
 
 
 class APContainer:
-    """A zipfile containing at least archipelago.json"""
-    version: int = container_version
-    compression_level: int = 9
-    compression_method: int = zipfile.ZIP_DEFLATED
-    game: Optional[str] = None
+    """A zipfile containing at least archipelago.json, which contains a manifest json payload."""
```

Comment:
Something like "subclasses extend the contents of the manifest" would be good here imo.